VisualizationBase/src/items/ViewItem.cpp

```diff
@@ -204,6 +262,11 @@ void ViewItem::determineChildren()
 				}
 		}
 	}
+	//Must make sure to add the necessary arrow layers here, as sometimes they are added
```

Comment:
Sometimes? but not always? What's the reason? We should find the root causes of such behaviors and not just implement workarounds that seem to do the trick.